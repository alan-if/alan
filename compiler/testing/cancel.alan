The l Isa location
  Has anEvent e.
End The l.

Event e
  "An event..."
End Event.

Start At l.
  Cancel e.
  Cancel anEvent of l.
  Cancel l.

