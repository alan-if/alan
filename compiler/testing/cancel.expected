########## cancel ##########


cancel.alan

   12.    Cancel l.
=====>           1

  *1*   331 E : Incompatible types in CANCEL statement. Event type required.


        1 error(s).
        No detected warnings.
        1 informational message(s).

